### PR TITLE
[cppgraphqlgen] Update to v4.1.1 for latest release

### PR DESCRIPTION
**Pull request by wravery**
**Describe the pull request**

- #### What does your PR fix?  
  Updates the latest version from v4.1.0 to v4.1.1.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All triplets are supported, but ci.baseline.txt still has an exception for x64-linux because the version of GCC is not new enough in the CI workflow.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: versions/c-/cppgraphqlgen.json [hunk 1]
```diff
@@ -10,6 +15,11 @@
       "version-semver": "4.0.0",
       "port-version": 0
     },
+    {
+      "git-tree": "c8481de46e09126a9b9c8a3a91a079ae61cd57bd",
+      "version-semver": "3.7.1",
+      "port-version": 0
+    },
```
Comment (JonLiu1993) [inline versions/c-/cppgraphqlgen.json:22]:
```suggestion
```

Comment (JackBoosY) [inline versions/c-/cppgraphqlgen.json:22]:
```suggestion
```
Please remove this line, this commit will not merged into master, so the users can't use this commit id to pull the related changes.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: versions/c-/cppgraphqlgen.json (hunk 1 above)
Comment (JonLiu1993) [reply versions/c-/cppgraphqlgen.json:22]: @wravery ,Thanks for your pr ,the git tree that is not pushed to the master cannot pull the corresponding commit

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/c-/cppgraphqlgen.json (hunk 1 above)
Comment (wravery) [reply versions/c-/cppgraphqlgen.json:22]: Ah, OK. Sounds like I can't really add the maintenance release version then, at least not without creating a separate port like `cppgraphqlgen3`.

**Review by JackBoosY — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**